app/src/main/java/ar/scacchipa/twittercloneapp/fragment/MainViewPagerAdapter.kt

```diff
@@ -0,0 +1,27 @@
+package ar.scacchipa.twittercloneapp.fragment
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import androidx.viewpager2.adapter.FragmentStateAdapter
+
+class MainViewPagerAdapter(
```

Comment:
If is not used, please remove the file

Reply:
It's true. The useless file was removed.